modules/apps/frontend-js/frontend-js-remote/paths/RemoteApplications.path

```diff
@@ -0,0 +1,41 @@
+<html>
+<head>
+<title>RemoteApplications</title>
+</head>
+
+<body>
+<table border="1" cellpadding="1" cellspacing="1">
+<thead>
+<tr><td rowspan="1" colspan="3">A11yTool</td></tr>
+</thead>
+
+<tbody>
+<tr>
+	<td>ADD_IFRAME</td>
+	<td>//button[contains(@class,'dropdown-item') and contains(., 'Add IFrame')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>THIS_FIELD_IS_REQUIRED</td>
+	<td>//div[contains(@class,'required') and contains(., 'This field is required.')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>IFRAME_NAME</td>
+	<td>//input[contains(@class,'form-control language-value')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>IFRAME_URL</td>
+	<td>//input[contains(@class,'field form-control error-field')]</td>
+	<td></td>
+</tr>
+<tr>
+	<td>PLEASE_ENTER_VALID_URL</td>
+	<td>//div[contains(@class,'alert alert-dismissible alert-danger')]</td>
```

Comment:
We want the name of the path to make sense with the element xpath. Here there's no mention of Please enter valid url. The xpath just has alert, alert-dismissible and alert-danger.

If we're choosing a generic danger type alert message we can reuse Message#ERROR or something similar in Message.path file.